src/app/components/PriceChart.jsx

```diff
@@ -151,35 +145,29 @@ const priceChartConfig = {
 export default compose(
   setDisplayName('PriceChart'),
   connect(createStructuredSelector({
-    priceChartData: getAllPriceCharts,
+    priceChartData: (state, { symbol }) => getPriceChartData(state, symbol)
```

Comment:
unsure about this variable name because its more like priceChartState or something like that. if you have any suggestions for renaming thatd be dope

Reply:
That name is fine